schemas/test/en/test_thank_you_census_communal_establishment.json

```diff
@@ -7,7 +7,7 @@
     "form_type": "C",
     "region_code": "GB-WLS",
     "title": "Census communal establishment test schema",
```

Comment:
Do we want to rename any of these schemas/titles? I think its probably fine without that but just wondering

Reply:
From Mebin's comment, I have remove this schema as census related schemas are not needed anymore